### DISCUSSION TITLE
( 質問）pwm_stopについて

### DISCUSSION THREAD

**Discussion by tsukada-shigeki** [toppers/users-forum#81] [TOPPERS Seminar]
竹内様
お世話になります。
１．タスク
　１）指令電圧（ｖｍ）入力タスク
　２）モータ逆起電圧（ｖｓ）入力タスク
　３）モータ速度制御出力タスク
２．成果
　速度制御している様です（怪しいですが）
　動画です。ご覧願います。
https://photos.app.goo.gl/ueyFuS6HSSsjuaE4A

３．質問
for(;;)の次に入っていたpwm_stopは不要と考えコメント化するとモータが動作しなくなります。出来ましたら推定原因をご教示願います。
![スクリーンショット (1120)](https://user-images.githubusercontent.com/106538497/180122202-c3d4c04a-99f9-4d72-b8db-12c2333b43ce.png)

**Comment by Ryohsuke-Takeuchi** [comment]
pwm_stopがないと、60ms毎にpwm_startが呼ばれます。2回目以降はエラーとなり停止ではないでしょうか？（確認していません）
pwm_startを実行して、停止したいとき、pwm_stopを呼ぶのが正解です。

**Reply by tsukada-shigeki** [reply]
竹内様
お世話になります。
メール拝受致しました。有難うございます。
＞pwm_startを実行して、停止したいとき、pwm_stopを呼ぶのが正解です。
⇒了解致しました。
お蔭様で基礎セミナー修了できました。
次のレベルのセミナー予定等ありましたらご教示願います。
